bundle-types/action.yml

```diff
@@ -0,0 +1,116 @@
+name: "Grafana Bundle Types"
+description: "Bundles a Grafana plugins types and pushes them to npm."
+
+outputs:
+  typesFilePath:
+    description: "The path to the built types file."
+    value: ${{ steps.set-outputs.outputs.typesFilePath }}
+
+inputs:
+  entryPoint:
+    description: "Location of types file to bundle."
+    required: false
+    default: "./src/types/index.ts"
+  tsConfig:
+    description: "A path to the tsconfig file to use when bundling types."
+    required: false
+    default: ""
+
+runs:
+  using: "composite"
+  steps:
```

Comment:
if we want this to only run in grafana repositories, what if we add a step that checks the repository org and if not grafana it fails the pipeline?

Reply:
Nice idea. Have added a step in https://github.com/grafana/plugin-actions/pull/40/commits/2aaa76ae80e795d4bcd01efd9cf43d1d511b5bc7.